napari/_vispy/text_utils.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from vispy.scene.visuals import Text
+
+
+def update_text(
```

Comment:
I'd probably drop the `text_` from the kwards like `text_color` and just make them `color` as function is called `update_text` it's probably unambigious what's going on.

Also needs doc string.

Reply:
I updated the signature and added a docstring.